examples/shapenet/README.md

```diff
@@ -91,6 +91,37 @@ The position will be in the center of the scene.
 We sample here five random camera poses, where the location is on a sphere with a radius of 2 around the object. 
 Each cameras rotation is such that it looks directly at the object and the camera faces upwards in Z direction.
 
+## HD5 Writer
+```
+"module": "writer.Hdf5Writer",
+    "config": {
+    "store_alpha": False,
+    "postprocessing_modules": {
+      "distance": [
+        {
+          "module": "postprocessing.TrimRedundantChannels",
+          "config": {}
+        }
+      ]
+    }
+}
+```
+To store alpha values in the hd5 files, use the store_alpha parameter.
+
+## RGB Renderer
+```
+"module": "renderer.RgbRenderer",
+  "config": {
+    "transparent_background": False,
+    "render_alpha": False,
```

Comment:
Can you explain in which case I would want to render_alpha but not use a transparent_background?

Furthermore, is the name `render_alpha` a bit confusing as alpha rendering is always supported, but this addition is only related to the background of the scene not the objects itself in the scene. 

I would maybe suggest only use `transparent_background`. I can not see a use case where people would want to render RGBA but not have a transparent background.

Reply:
This relates to the above, I'll take your advice and refactor to only use transparent_background.